content/docs/deployment.md

```diff
@@ -208,6 +239,34 @@ Shards and replicas are some configuration values to take special attention to,
 index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
 more information about choosing how many shards should be chosen for optimization.
 
+### Kafka
+Supported in Jaeger since 1.6.0
+Supported Kafka versions: 0.8+
+
+Considering the Kafka storage backend implementation only supports writing, you will need to use the multiple storage types feature.
```

Comment:
Add: Writing to Kafka is particularly useful for building post-processing data pipelines.